packages/grafana-data/src/dataframe/CircularDataFrame.ts

```diff
@@ -10,6 +10,8 @@ interface CircularOptions {
 /**
  * This dataframe can have values constantly added, and will never
  * exceed the given capacity
+ *
+ * @deprecated use standard DataFrame
```

Comment:
should the recommendation here be to use `StreamingDataFrame` ?

also: https://github.com/grafana/grafana/blob/main/docs/sources/developers/plugins/build-a-streaming-data-source-plugin.md